### PR TITLE
Document `NC_MAX_ATTACHMENTS_ALLOWED`

### PR DESCRIPTION
**Pull request by salim-b**
## Change Summary

`NC_MAX_ATTACHMENTS_ALLOWED` was introduced in https://github.com/nocodb/nocodb/pull/4820.

## Change type

- [x] docs: (changes to the documentation)

### CODE REVIEW

File: packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md [hunk 1]
```diff
@@ -59,8 +59,9 @@ For production use-cases, it is **recommended** to configure
...
-| NC_ADMIN_EMAIL                     | For updating/creating super admin with provided email and password                                                                                                                                                                          |                                                                                                |
 | NC_ATTACHMENT_FIELD_SIZE           | For setting the attachment field size(in Bytes)                                                                                                                                                                                             | Defaults to 20MB                                                                               |
+| NC_MAX_ATTACHMENTS_ALLOWED         | Maximum Number of attachments per cell                                                                                                                                                                                                      |                                                                                                |
```
Comment (wingkwong) [inline packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: FYI. `NC_MAX_ATTACHMENTS_ALLOWED` is not used for OSS but EE version.

Comment (salim-b) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: Ah, so it has no effect on the official `nocodb/nocodb` Docker image?

Does a maximum number of attachments also apply if an external object storage provider (e.g. Backblaze B2) is configured?

Comment (wingkwong) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: oh wait. seems i might mix up something. lemme check with the team first and get back to you later.

**Review by wingkwong — APPROVED** [review]

File: packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md (hunk 1 above)
Comment (wingkwong) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: Confirmed with the team. This should be included in OSS as well.

Comment (salim-b) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: Glad to hear! :+1:

Comment (salim-b) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: @wingkwong Does the limit apply regardless whether internal storage or external object storage (S3) is used?

Comment (wingkwong) [reply packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md:63]: Correct. It's just the limit of files to be uploaded in one cell.

**Pull request merged**